easybuild/easyconfigs/v/VMD/VMD-1.9.4a51-foss-2020b.eb

```diff
@@ -0,0 +1,59 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'VMD'
+version = '1.9.4a51'
+
+homepage = 'https://www.ks.uiuc.edu/Research/vmd'
+description = """VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular
+ systems using 3-D graphics and built-in scripting."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = [
+    'https://www.ks.uiuc.edu/Research/vmd/vmd-1.9.4/files/alpha/',
+    'https://www.ks.uiuc.edu/Research/vmd/vmd-%(version)s/files/final',
+    'https://webclu.bio.wzw.tum.de/stride/'
+]
+sources = [
+    'vmd-%(version)s.src.tar.gz',
+    {'filename': 'stride.tar.gz', 'extract_cmd': "tar -C vmd-%(version)s/lib/stride -xf %s"},
+]
+patches = [
+    ('VMD-1.9.3_plugins.patch'),
+    ('VMD-1.9.3_surf_Makefile.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_surf_bad_printfs.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_stride_Makefile.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_stride_MAX_AT_IN_RES.patch', 'vmd-%(version)s'),
+    ('VMD-%(version)s_configure.patch', 'vmd-%(version)s'),
+    ('VMD-%(version)s_extra_colors.patch', 'vmd-%(version)s'),
+]
+checksums = [
+    'b1c40b21111f5bab56d43d5e442c468d327159b07915af2ec175ba6b12842e5c',  # vmd-1.9.4a51.src.tar.gz
+    '51a8bc2988bb184bd08216124f61725225bb1a6f563bdf8cd35154cb5d621c1a',  # stride.tar.gz
+    '85760d6ae838e2b09801e34b36b484532383f7aaf2e8634b3ef808002a92baa3',  # VMD-1.9.3_plugins.patch
+    'd5cfa88064b7cffbc75accd69707d4e45fda974e8127de9ab606fdad501bd68a',  # VMD-1.9.3_surf_Makefile.patch
+    'f3c2a8c155e38db8e644cee6a01f6beaea5988e72ac74cde26b71670b151cc34',  # VMD-1.9.3_surf_bad_printfs.patch
+    'eb194ac0d8c086b73f87b29f7d732687f902431b1cdfa139c090401fefdee51e',  # VMD-1.9.3_stride_Makefile.patch
+    'eff1ca00cec637a6c8a156b2fb038e078d1835ba0eb15a571ed820bca5a866d9',  # VMD-1.9.3_stride_MAX_AT_IN_RES.patch
+    '53af80c866a9221559b4882bfc9e7de8a5a5388b4a2c5cc91d6d55100660dd2f',  # VMD-1.9.4_configure.patch
+    '253eba282b570eb00e4764f46f77fd5ca898d10360d5707dd50ad1f14615af80',  # VMD-1.9.4_extra_colors.patch
+]
+
+dependencies = [
+    ('Tcl', '8.6.10'),
+    ('Tk', '8.6.10'),
+    ('FLTK', '1.3.5'),
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Tkinter', '%(pyver)s'),
+    ('Mesa', '20.2.1'),
+    ('netCDF', '4.7.4'),
+    ('FFmpeg', '4.3.1'),
+    ('ImageMagick', '7.0.10-35'),
+    ('ACTC', '1.1'),
+    ('zlib', '1.2.11'),
+    ('libpng', '1.6.37'),
```

Comment:
please add POV-Ray 3.0.7.8 as dep. It's needed to make Movies

Reply:
Done. I will do some additional build tests on my machine.